openapi/index_openapi.json

```diff
@@ -1553,7 +1606,7 @@
           "representation": {
             "title": "Representation",
             "type": "string",
-            "description": "a string with the part of the URL that follows the base URL. Example: '/structures?'"
+            "description": "A string with the part of the URL following the versioned or unversioned base URL that serves the API.\nQuery parameters that have not been used in processing the request MAY be omitted.\nIn particular, if no query parameters have been involved in processing the request, the query pary of the URL MAY be excluded.\nExample: `/structures?filter=nelements=2`"
```

Comment:
```suggestion
            "description": "A string with the part of the URL following the versioned or unversioned base URL that serves the API.\nQuery parameters that have not been used in processing the request MAY be omitted.\nIn particular, if no query parameters have been involved in processing the request, the query part of the URL MAY be excluded.\nExample: `/structures?filter=nelements=2`"
```

Reply:
Well spotted, this must have been fixed in a recent spec update?

Reply:
Aaaah, its actually a typo in the new schemas :S

Reply:
I think the best action here is to get this PR merged once we're completely happy with it, then make a PR that replaces the current schemas with our 100% auto-generated ones again. We can then probably release 1.0.1 of the specification after a week or two once people have implemented stuff (assuming there are no other breaking fixes).